src/mlpack/methods/ann/layer/sequential.hpp

```diff
@@ -54,6 +74,7 @@ class Sequential
    * Create the Sequential object using the specified parameters.
    *
    * @param model Expose the all network modules.
+   * @param residual Use the object as a residual block.
```

Comment:
I think this line can be shifted to the template definition.

Reply:
Ahh, I forgot about that.